projects/backend/src/main/java/backend/service/impl/AttendanceServiceImpl.java

```diff
@@ -0,0 +1,58 @@
+package backend.service.impl;
+
+import backend.domain.Attendance;
+import backend.domain.Lecture;
+import backend.domain.User;
+import backend.repositories.AttendancesRepository;
+import backend.service.AttendanceService;
+
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+public class AttendanceServiceImpl implements AttendanceService {
+
+  private final AttendancesRepository attendancesRepository;
+
+  @Autowired
+  public AttendanceServiceImpl(AttendancesRepository attendancesRepository) {
+    this.attendancesRepository = attendancesRepository;
+  }
+
+  @Override
+  public Attendance get(String attendanceId) {
+    return attendancesRepository.findById(attendanceId)
+        .orElse(null);
```

Comment:
It looks that ```AttendanceService``` was dummy constructed, it should return an ```Optional```. Ok, but we should add a task on trello regarding addnig ```optional``` to the service layer response